sechub-pds-solutions/multi/cluster_object_storage.svg

```diff
@@ -0,0 +1,240 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am not happy to always copy` cluster_object_storage.svg`in every PDS solution. It is normally always the same SVG in every PDS solution. So always duplicated.
And for maintainance - maybe we improve the picture some time - this will be much effort (we have to change all of them).

The SVG is used by asciidoc files. so we should accomplish a common folder for common pds-solution graphics.
E.g. `sechub-pds-solutions/common/gfx`